### PR TITLE
Use 0.19 release of toml_edit

### PR DESCRIPTION
**Pull request by matze**
Finally had the time and essentially port #29 to toml_edit 0.19. All tests pass (on Linux), so there's that.

### CODE REVIEW

**Comment by matze** [comment]
Not sure about the Windows failure. I would actually expect no `\r` to exist unlike what the test asserts.

File: src/fmt.rs [hunk 1]
```diff
@@ -251,12 +290,9 @@ mod test {
...
-            input.replace("\r\n", "\n"),
-            toml.to_string_in_original_order().replace("\r\n", "\n")
-        );
+        assert_eq!(input.replace("\r\n", "\n"), toml.to_string().replace("\r\n", "\n"));
         #[cfg(not(target_os = "windows"))]
-        assert_eq!(input, toml.to_string_in_original_order());
+        assert_eq!(input, toml.to_string());
     }
 
     #[cfg(target_os = "windows")]
```
Comment (DevinR528) [inline src/fmt.rs:298]: Sorry github won't let me click on line 305 should be
```
"[package]\nname = \"priv-test\"\nversion = \"0.1.0\"\nedition = \"2021\"\nresolver = \"2\"\n# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html\n\n[dependencies]\nstructopt = \"0.3\"\n"
```

(Remove the `\r` as you suggested)

File: src/fmt.rs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 use std::str::FromStr;
 
-use crate::toml_edit::{Document, Item, Table, Value};
+use toml_edit::{Document, Item, Table, Value};
 
 /// The config file for formatting toml after sorting.
 ///
```
Comment (DevinR528) [inline src/fmt.rs:6]: line 57 above the doc comments for `pub crlf: bool` could you add
```
// NOTE: this is only used in main, fmt doesn't set the line endings
```
(I forget this every time I look through this code base) Thanks.

**Review by DevinR528 — CHANGES_REQUESTED** [review]
The `fixtures/` folder and all things within are junk, they were only for `toml_edit` tests you can leave that as a separate commit if that works.

Thanks again!

**Comment by matze** [comment]
I added your commits, thanks for having a look on Windows as well :-)

**Comment by SimenB** [comment]
Ping 😀

**Comment by jplatte** [comment]
An upgrade was merged in #29, I'll upgrade again now.

**Pull request closed without merging**